routes/register.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express')
 const router = express.Router()
 
-const { register } = require('../services/auth')
+const { register } = require('../controllers/register')
 
 router.post('/register', register)
```

Comment:
```suggestion
router.post('/register', post)
```
Then this fixes to the above